cluster/gce/upgrade.sh

```diff
@@ -328,35 +328,45 @@ node_prereqs=false
 local_binaries=false
 env_os_distro=false
 
-while getopts ":MNPlho" opt; do
-  case ${opt} in
-    M)
+args=$(getopt -o MNPlho -- "$@")
```

Comment:
This doesn't work on my mac. If I echo $args right after this line when running `upgrade.sh -h` I see `-- MNPlho -- -h` and the first iteration through the while loop has `$1` set to `--`. And no arguments are parsed.